github-pullrequest-plugin/src/test/java/org/jenkinsci/plugins/github/pullrequest/trigger/check/BranchRestrictionFilterTest.java

```diff
@@ -32,6 +33,7 @@ public void shouldNotFilterWithNotEnabledBranchRestriction() throws Exception {
         assertThat("when br is null", withBranchRestriction(tlRule.getListener(), null).apply(null), is(true));
     }
 
+    @Ignore("TODO needs triage")
```

Comment:
what happen with this tests?

Reply:
@KostyaSha The mocks needed to be updated to adapt to changes in third-party libraries. I have done so in commit db330dc and repeated my testing by running `mvn clean verify` successfully on Java 8.

Reply:
jfyi we can go forward with java11

Reply:
The mocking code will need significant updating for Java 11. I am not prepared to do that work at this time.